ionic_user_interface/src/composables/useCanvasWidth.ts

```diff
@@ -0,0 +1,27 @@
+import { Ref, ref } from 'vue';
+import { debounce } from 'lodash';
+
+export function useCanvasWidth(): {
+  canvasWidth: Ref<number>;
+  updateCanvasWidth: () => void;
+} {
+  const canvasWidth = ref(0);
```

Comment:
Maybe we should set a default of 700px or something

Reply:
I think this is fine because they won't be shown at all if empty